src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/_update.py

```diff
@@ -19,7 +19,7 @@ class Update(AAZCommand):
 
     :example: Updates a Service Bus Namespace
         az servicebus namespace update --resource-group myresourcegroup --name mynamespace --tags tag=value
-        az az servicebus namespace update --name mynamespace --resource-group myresourcegroup --sku Basic
+        az servicebus namespace update --name mynamespace --resource-group myresourcegroup --sku Basic
```

Comment:
This should also be applied to aaz repo since it's auto generated code. cc @necusjz @kairu-ms for guide